CONTRIBUTING.md

```diff
@@ -1,13 +1,18 @@
 # Contributing Guidelines
 
+## Required tooling
+
+- [Helm 3](https://helm.sh/docs/intro/install/): Most popular Kubernetes templating tool.
+- [Helm Schema-gen](https://github.com/mihaisee/helm-schema-gen.git): can be installed using `helm plugin install https://github.com/mihaisee/helm-schema-gen.git`. This tool is used to generate the json schema of the helm chart
+
 ## Upgrading
 
 * change the `promtail` upstream version in Chart dependencies (`helm/promtail/Chart.yaml`)
 * run `helm dependency update helm/promtail` to update the Chart.lock file
 * re-generate `helm/promtail/values.schema.json`:
   * `helm schema-gen helm/promtail/values.yaml > helm/promtail/values.schema.json` to re-generate the file.
   * `sed -i 's/"type": "null"/"type": ["string", "null"]/g' helm/promtail/values.schema.json` to accept strings for all null values.
-* update the link in the `Configuration` section of the Readmeto point to the new tag configuration.
+* update the link in the [`Configuration`](./README.md#configuration) section of the Readmeto point to the new tag configuration.
```

Comment:
Readmeto => Readme to
;-)

Reply:
You know you can suggest changes ? 

![image](https://user-images.githubusercontent.com/2787548/214797364-02ee7d69-d5a2-4a35-92c1-cce33268c00e.png)

![image](https://user-images.githubusercontent.com/2787548/214797396-29772cc5-ef31-4dd8-8729-cb321d1a1151.png)

Try it this is awesome :)

Reply:
I applied it. Can you review one last time?